templates/vue/src/components/Toolbar/HelpButton.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+     <button aria-label="get help" @click="open">
+        <tapestry-icon icon="question-circle" />
+        <help-modal
+            :show="helpModalOpen"
+            @close="close" 
+        ></help-modal>
+     </button>
+</template>
+
+<script>
+import HelpModal from "@/components/modals/HelpModal"
+import TapestryIcon from "@/components/common/TapestryIcon"
+import { names } from "@/config/routes"
+import client from "@/services/TapestryAPI"
+
+
+export default {
+  components: {
+    HelpModal,
+    TapestryIcon,
+  },
+  computed: {
+ //   ...mapState(["settings", "nodes", "rootId"]),
```

Comment:
Make sure not to leave commented out code - or if you do, please prepend it with a comment saying why it's commented out